baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +608,26 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+/**
+ * This method is for decoding operations.
+ * @param {String} name
+ *   The operation name that will be passed.
+ * @returns {Promise} - The promise which resolves to an array.
+ *   The first element of the array is an object representing the result of operations.
+ *   The second element of the array is an object representing the metadata of operations.
+ *
+ * @code sample:
+ *   const decodedOperation = await waitDecodeLRO(name);
+ *   console.log(decodedOperation.result);
+ *   console.log(decodedOperation.metadata);
+ *
+ */
+  async waitDecodeLRO(name: string): Promise<LROperation<{} | null, Metadata | null>>{
```

Comment:
We still need to invent a better name. @JustinBeckwith do you still have questions about why we need this one?

Reply:
loop @bcoe as well for some suggestions :)

Reply:
we have `check*progress` and `*AwaitResult` name for now.